api/api.yml

```diff
@@ -0,0 +1,191 @@
+  - name: BlockModel
+    x-displayName: Block
+    description: |
+      A Block contains information relevant to a particular point in time over which the network may achieve consensus. Blocks are mined as part of the consensus mechanism by elected miners and then included in a Tipset.
+      <SchemaDefinition schemaRef="#/components/schemas/Block"/>
+  - name: Blocks
+  - name: ChannelModel
+    x-displayName: Channel
+    description: |
+      <SchemaDefinition schemaRef="#/components/schemas/Channel"/>
+  - name: DealModel
+    x-displayName: Deal
+    description: |
+      <SchemaDefinition schemaRef="#/components/schemas/Deal"/>
+  - name: ExecutedMessageModel
+    x-displayName: ExecutedMessage
+    description: |
+      <SchemaDefinition schemaRef="#/components/schemas/ExecutedMessage"/>
+  - name: MessageModel
+    x-displayName: Message
+    description: |
+      <SchemaDefinition schemaRef="#/components/schemas/Message"/>
+  - name: Messages
+  - name: Node
+  - name: NodeModel
+    x-displayName: Node
+    description: |
+      <SchemaDefinition schemaRef="#/components/schemas/Node"/>
+  - name: TipsetModel
+    x-displayName: Tipset
+    description: |
+      <SchemaDefinition schemaRef="#/components/schemas/Tipset"/>
+  - name: Tipsets
+  - name: VoucherModel
+    x-displayName: Voucher
+    description: |
+      <SchemaDefinition schemaRef="#/components/schemas/Voucher"/>
+x-tagGroups:
+  - name: Chain Resources
+    tags:
+      - Tipsets
+      - Blocks
+      - Messages
+  - name: Actor Endpoints
+    tags:
+      - Actors
+  - name: Channel Endpoints
+    tags:
+      - Channels
+  - name: Node Control Endpoints
+    tags:
+      - Node
+      - Address
+      - Pieces
+  - name: Storage Market Endpoints
+    tags:
+      - Asks
+      - Deals
+  - name: Models
+    tags:
+      - ActorModel
+      - AskModel
+      - BlockModel
+      - BlockHeaderModel
+      - ChannelModel
+      - DealModel
+      - ExecutedMessageModel
+      - MessageModel
+      - NodeModel
+      - TipsetModel
+      - VoucherModel
+info:
+  title: Filecoin JSON REST API
+  description: Filecoin JSON REST API Description
+  version: 0.1.0
+  contact: 
+    name: Filecoin JSON REST API SPEC
+    email: erik.ingenito@protocol.ai
+servers:
+- url: http://127.0.0.1:3453/api/chain/v0
+security:
+  - 'Bearer Token Authentication': []
+components:
+  securitySchemes:
+    'Bearer Token Authentication':
+      type: http
+      scheme: Bearer Token
+      description: |
+        V0 of this API supports all or nothing authentication and authorization suitable for use by the owner/operator of the current node, and nothing else. Callers of the API must provide a bearer token included in the Authorization header of each request. This token should be provided by the node for use by the caller via some out of band mechanism (like a file created on the filesystem upon initialization or upon startup).
```

Comment:
"Callers of the API may provide". Reword this to make it clear that authentication is optional by the server, but if it wants to do bearer authentication, this is the standard way. Implementations could provide other site-specific auth methods too.